pkg/pipeline/pipeline.go

```diff
@@ -24,6 +24,7 @@ const (
 	AssetTypePostgresQuery        = AssetType("pg.sql")
 	AssetTypeRedshiftQuery        = AssetType("rs.sql")
 	AssetTypeMsSQLQuery           = AssetType("ms.sql")
+	AssetTypeSynapseQuery         = AssetType("sy.sql")
```

Comment:
I don't think `sy.sql` is obvious for synapse, shall we go with `synapse.sql`, @sabrikaragonen what are your thoughts?

Reply:
i agree, sy is very weird